tasks/main.yml

```diff
@@ -1 +1,23 @@
 # role tasks
+- name: Upgrade kexec-tools to latest
+  package:
+    name: "{{ kdump_version }}"
+    state: present
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Edit kdump config file
+  template:
+    src: ../templates/kdump_conf
```

Comment:
You can remove the `../templates/` from this as, by default, the `template` module will look in the `templates` folder of the current role.